invenio_rdm_records/resources/serializers/dublincore/schema.py

```diff
@@ -0,0 +1,189 @@
+        for f in fundings:
+            award = f.get('award')
+            if award:
+                rels.append(
+                    u'{0}:{1}'.format(award['scheme'], award['identifier'])
+                )
+
+            funder = f.get('funder', {})
+            if funder:
+                rels.append(
+                    u'{0}:{1}'.format(funder['scheme'], funder['identifier'])
+                )
+
+        # Alternate identifiers
+        for a in obj['metadata'].get('alternate_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        # Related identifiers
+        for a in obj['metadata'].get('related_identifiers', []):
+            rels.append(u'{0}:{1}'.format(a['scheme'], a['identifier']))
+
+        return rels
+
+    def get_rights(self, obj):
+        """Get rights."""
+        rights = []
+        for right in obj['metadata'].get('rights', []):
+            rights.append(right['title'])
+
+            license_url = right.get('link')
+            if license_url:
+                rights.append(license_url)
+
+        return rights
+
+    def get_dates(self, obj):
+        """Get dates."""
+        dates = [obj["metadata"]["publication_date"]]
+
+        if obj['access']['status'] == 'embargoed':
+            dates.append(obj['access']['embargo']['until'])
+
+        return dates or missing
+
+    def get_descriptions(self, obj):
+        """Get descriptions."""
+        metadata = obj['metadata']
+        descriptions = []
+
+        description = metadata.get("description")
+        if description:
+            descriptions.append(description)
+
+        additional_descriptions = metadata.get("additional_descriptions", [])
+        for add_desc in additional_descriptions:
+            descriptions.append(add_desc['description'])
+
+        return descriptions
+
+    def get_subjects(self, obj):
+        """Get subjects."""
+        metadata = obj['metadata']
+        subjects = []
+        subjects.extend(metadata.get('keywords', []))
+        subjects.extend(
+            (
+                s['subject']
+                for s in metadata.get('subjects', [])
+                if s.get('subject')
+            )
+        )
+        return subjects
+
+    def get_publishers(self, obj):
+        """Get publishers."""
+        return [obj['metadata']['publisher']]
+
+    def get_contributors(self, obj):
+        """Get contributors."""
+        return [
+            c['person_or_org']['name']
+            for c in obj['metadata'].get('contributors', [])
+        ]
+
+    def get_types(self, obj):
+        """Get resource type."""
+        type = obj['metadata']['resource_type']
+        return [type['id']]
+
+    def get_sources(self, obj):
+        """Get sources."""
+        items = []
+        return missing
+
+    def get_languages(self, obj):
+        """Get language."""
+        metadata = obj["metadata"]
+        languages = metadata.get("languages")
```

Comment:
Simpler, and consistent with the rest of the code :) 👼 
```suggestion
        languages = obj["metadata"].get("languages")
```